bool request
---
float32[] point_path
int32 dim